sdk/loadtestservice/Azure.Developer.LoadTesting/CHANGELOG.md

```diff
@@ -1,14 +1,33 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2022-12-01)
```

Comment:
Please update to the correct release date.

Reply:
This is done automatically by the [Prepare-Release](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#preparing-a-new-library-release) script